pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -94,6 +94,10 @@ type ControllerConfigSpec struct {
 
 	// proxy holds the current proxy configuration for the nodes
 	Proxy *configv1.ProxyStatus `json:"proxy"`
+
+	// infra holds the infrastructure details
+	// TODO this makes platform redundant as everything is contained inside Infra.Status
+	Infra configv1.Infrastructure
```

Comment:
Pointer?

Reply:
Why a pointer? are the values changing in a differnet place?

Reply:
maintaining the pattern with other infra object and useful to differentiate between set/unset (not that it's needed now, but still)

Reply:
So are we going to use that for setting value to  it? Is that the generated code maybe? (help me to get that cause I'd like to understand it)